mllib/src/main/scala/org/apache/spark/ml/recommendation/ALS.scala

```diff
@@ -1229,16 +1229,19 @@ object ALS extends DefaultParamsReadable[ALS] with Logging {
     // elements distributed as Normal(0,1) and taking the absolute value, and then normalizing.
     // This appears to create factorizations that have a slightly better reconstruction
     // (<1%) compared picking elements uniformly at random in [0,1].
-    inBlocks.map { case (srcBlockId, inBlock) =>
```

Comment:
what about just `mapValues`?

Reply:
Agree except that `srcBlockId` is used below

Reply:
ah, I see!

Reply:
I see